templates/ts-apollo-fullstack/codegen.yml

```diff
@@ -0,0 +1,13 @@
+schema: ./server/src/schema/schema.graphql
+generates:
+  ./client/src/generated-types.tsx:
+    config:
+      withComponent: false
```

Comment:
@darahayes @kingsleyzissou 

Codegen resolvers nicely problem of hooks vs hoc by giving people flags. 
What we get from codegen is fully featured data layer. No need to do anything apart from using hooks.

Reply:
Yeah perhaps generating the hooks could be a config option.

Reply:
Yep.. That is already there - what we need to do is to just call offline mutate in generated-types.ts - which is small extension to existing apollo plugin. It will be much easier than writing that from scratch, but also tie us to codegen